waspc/data/Generator/templates/server/src/dbClient.js

```diff
@@ -12,6 +13,41 @@ const createDbClient = () => {
   const prismaClient = new Prisma.PrismaClient()
 
   {=# isAuthEnabled =}
+
+  prismaClient.$use(async (params, next) => {
+    // Ensure strong plaintext password. Must come before hashing middleware.
+    // Throws an EntityValidationError on the first validation that fails.
+    if (params.model === '{= userEntityUpper =}' && params.action === 'create') {
```

Comment:
I noticed in middleware below that it is observing multiple actions: create, update ... .
Should we do the same thing here? What if somebody modifies the password? In that case these validators will not run, which I guess is an issue?

Reply:
Ah, great point, `update*` is an issue too! Thanks! 👍🏻

Reply:
oh wow, well spotted @Martinsos !

Reply:
This has been addressed here: https://github.com/wasp-lang/wasp/pull/376/commits/3ffbc62c75cf4a43363b90fc8c2b1480309891ba Thanks!